src/common/check_connectivity.cpp

```diff
@@ -60,19 +68,15 @@ Network::Network(void)
         throw std::bad_alloc();
     }
 
-    if ((dnsbase = evdns_base_new(evbase, 0)) == NULL) {
+    if ((dnsbase = evdns_base_new(evbase, 1)) == NULL) {
```

Comment:
Ah, yes, sorry. This is correct and was the original objective of the PR. I'll merge this as soon as I can test it, hopefully by end of week.